README.md

```diff
@@ -68,7 +68,7 @@ curl -fLo ~/.var/app/io.neovim.nvim/data/nvim/site/autoload/plug.vim \
 
 ```powershell
 iwr -useb https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim |`
-    ni "$env:LOCALAPPDATA/nvim-data/site/autoload/plug.vim" -Force
+    ni "$(@($env:XDG_DATA_HOME, $env:LOCALAPPDATA)[$null -eq $env:XDG_DATA_HOME])/nvim-data/site/autoload/plug.vim" -Force
```

Comment:
`stdpath('data')` is `~/.local/share/nvim` on my machine. Neovim does not use `nvim-data` on Unix. See `:h standard-path`.

Reply:
When I set XDG_DATA_HOME it creates a nvim-data directory.

Reply:
On Windows?

Reply:
Yes

Reply:
```
PS C:\Users\dylha\OneDrive\Config> echo $Env:XDG_DATA_HOME
C:\Users\dylha\OneDrive\Config\nvim
PS C:\Users\dylha\OneDrive\Config> ls C:\Users\dylha\OneDrive\Config\nvim


    Directory: C:\Users\dylha\OneDrive\Config\nvim

Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
la---           12/2/2020 12:25 PM                nvim-data

```

Reply:
Related: https://github.com/neovim/neovim/issues/9910

Reply:
I wouldn't call this a bug? I'm only explaining why I reserved the `/nvim-data` part in the script.

Reply:
Not saying this is a bug or anything. Posted it as a reference where I learned about how XDG_DATA_HOME is used on Neovim on Windows.